code/__DEFINES/wounds.dm

```diff
@@ -77,10 +77,12 @@ GLOBAL_LIST_INIT(global_all_wound_types, list(/datum/wound/blunt/critical, /datu
 // ~random wound balance defines
 /// how quickly sanitization removes infestation and decays per second
 #define WOUND_BURN_SANITIZATION_RATE 0.075
-/// how much blood you can lose per second per slash max. 4 is a LOT of blood for one cut so don't worry about hitting it easily
-#define WOUND_SLASH_MAX_BLOODFLOW 4
+/// how much blood you can lose per tick per slash max.
```

Comment:
Why per tick? Per second is always easier to think about

Reply:
The proc only runs every tick, but I spoke to you regarding the per second/per tick thing on discord